Felienne/hedy
# Fix #1636

**Description**

Fix the issue where `chart.js` is called on every page, resulting in some undesirable decency errors. By storing some code inside a function that is only called when we call the dedicated page the error is fixed.

**Fix for**
This PR fixed #1636.

**How to test**
Load any page on alpha and on this branch and navigate to your browser console. Notice that the `chart.js` error is no longer there. (The other errors are for a later moment to solve).

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
